src/react/src/components/DashboardClaimsDetails.jsx

```diff
@@ -149,31 +149,24 @@ function DashboardClaimsDetails({
                     }
                 />
                 <InfoSection
-                    label="Facility Parent Company / Supplier Group"
-                    value={(() => {
-                        const parentCompanyName = get(
-                            data,
-                            'facility_parent_company.name',
-                            null,
-                        );
-
-                        if (!parentCompanyName) {
-                            return '';
-                        }
-
-                        const profileLink = makeProfileRouteLink(
-                            get(data, 'facility_parent_company.id', null),
-                        );
-
-                        return (
-                            <Link to={profileLink} href={profileLink}>
-                                {parentCompanyName}
-                            </Link>
-                        );
-                    })()}
+                    label="Production Location's Website"
+                    value={
+                        data.facility_website && (
```

Comment:
This can be refactor like this to follow readability and separation of concers concept:

```
const FacilityWebsiteLink = ({ website }) => {
    if (!website) return null;

    const formattedURL = addProtocolToWebsiteURLIfMissing(website);

    return (
        <a href={formattedURL} target="_blank" rel="noopener noreferrer">
            {website}
        </a>
    );
};

const DashboardClaimsDetails = ({ data }) => (
    ....
    <InfoSection
        label="Production Location's Website"
        value={<FacilityWebsiteLink website={data.facility_website} />}
    />
    ....
);

```

Reply:
I have refactored DashboardClaimsDetails but slightly differently because there were several places with repetitive code so I extracted the repetitive logic into helper functions.